src/components/ListOfMembers.jsx

```diff
@@ -1,12 +1,20 @@
 import Button from '@mui/material/Button';
+import { Link } from 'react-router-dom';
 
 export default function ListOfMembers(props) {
+  
+  const handleMemberToView = (event, member) => { 
+    console.log("handleMemberToView: ", member)
```

Comment:
Consider removing the console log if it's no longer needed :)

Reply:
will delete as going forward
thanks